src/xdocs/writingjavadocchecks.xml.vm

```diff
@@ -558,11 +558,123 @@ JAVADOC -> JAVADOC [0:0]
         </td>
       </tr>
     </table>
-    </section>
 
-    <section name="Boolean flag indicating the presence of unclosed HTML tags">
-      Not implemented yet, will be named "hasUnclosedTag". See <a href="https://github.com/checkstyle/checkstyle/issues/3311">Github Issue #3311</a>.
-      For expected behavior please read <a href="#Javadoc_parser_behavior_for_current_HTML_version_and_new_HTML_version">Javadoc parser behavior</a> section.
+    <p>
+        Checks can also be configured to log violation on encoutering non-tight HTML tags.
+        <tt>violateExecutionOnNonTightHtml</tt> property can be used for this purpose in the checks
+        that support it. A custom check needs to extend <tt>AbstractJavadocCheck</tt> for having
+        this functionality readily available. Do note that a check which has this property set to
+        true will log all violations upon finding non-tight HTML tags and then proceed to the
+        next javadoc. The following example illustrates how to use this property.
+    </p>
+
+    <p>
+    Input:
+    </p>
+
+    <source><![CDATA[
+/**
+  * <body>
+  * <p> This class is only meant for testing. </p>
+  * <p> In html, closing all tags is not necessary.
```

Comment:
please make line like "this P tag is not closed"

Reply:
Done.